sql/core/src/main/scala/org/apache/spark/sql/catalog/Catalog.scala

```diff
@@ -33,36 +33,37 @@ import org.apache.spark.storage.StorageLevel
 abstract class Catalog {
 
   /**
-   * Returns the current default database in this session.
+   * Returns the current database (namespace) in this session.
```

Comment:
there is a different way to refer to here: schema. 

Are we have decided to use namespace in Spark?

Reply:
`namespace` is more like the official name. database/schema is only for the hive catalog. We can change `database` to `database/schema` though.

Reply:
sounds good. Just wanted to confirm that we don't miss anything obvious.